src/java/org/apache/cassandra/cql3/Lists.java

```diff
@@ -154,6 +155,8 @@ public Literal(List<Term.Raw> elements)
 
         public Term prepare(String keyspace, ColumnSpecification receiver) throws InvalidRequestException
         {
+            if (receiver.type instanceof VectorType)
```

Comment:
I honestly hate this... prob should have a `ListOrVector` class as we don't know until `prepare`?

Reply:
yeah, ill make that change... I think this only impacts Parser.g but will confirm

Reply:
pushed this change.  Can't use `Arrays` as classes in this package freak out as they were using JDK `Arrays`.  Since its only a single `Literal` we want went with `ArrayLiteral`